code/game/objects/items/stacks/rods.dm

```diff
@@ -115,6 +115,7 @@
 		return
 	else //End click drag construction, create grille
 		returnToPool(active)
+		active = null
```

Comment:
if you can make a grille, this isn't necessary

Reply:
okay so I'm not entirely sure about this line and the other line
what makes it so that these ones, specifically, aren't necessary?

Reply:
its because of the way it is

Reply:
🤔